.env

```diff
@@ -0,0 +1 @@
+PORT=3000
```

Comment:
Make sure to add this to your .gitignore:
`.env`
`.env.example`
`.env.local`
`.env.staging`

There is also a command that you can run in your terminal that will delete what you have pushed up which is:
`git rm .env --cached`
and then for your commit you can do something like:
`git commit -m "stopped tracking .env"`

Reply:
ok thank you!